components/script/dom/bindings/trace.rs

```diff
@@ -492,6 +495,8 @@ unsafe_no_jsmanaged_fields!(RenderApiSender);
 unsafe_no_jsmanaged_fields!(ResourceFetchTiming);
 unsafe_no_jsmanaged_fields!(Timespec);
 unsafe_no_jsmanaged_fields!(HTMLMediaElementFetchContext);
+unsafe_no_jsmanaged_fields!(StyleSheetListOwner);
```

Comment:
I don't think this is sound. `StyleSheetListOwner` can totally have jsmanaged fields, right? Why do you need this?